app/api/save-address/route.js

```diff
@@ -0,0 +1,29 @@
+import { NextResponse } from "next/server";
+import { queryPromise, pool } from "@/utils/sql";
+
+
+export async function POST(request) {
```

Comment:
is it possible to change these functions name to SAVE_ADDRESS

Reply:
The same thing applies here